src/Atoms/Inputs/Input.js

```diff
@@ -134,13 +136,15 @@ const Input = ({ className, input, type, ...rest }: PropsType) => {
           }${rest.error || ''}`}
         </InputLabel>
         <InputWrapper {...input} {...rest} type={type} />
+        {rest.renderSuffix && rest.renderSuffix()}
```

Comment:
We should probably pass a few things to `renderSuffix`. `value`, `disabled` and `error` should be enough. That way the suffix can react to those props.